.azurepipelines/CargoMakeArtifact.yml

```diff
@@ -0,0 +1,26 @@
+## @file
+# Azure Pipeline to build and publish Cargo Make in a format supported by edk2-pytool-extension's external dependency.
+#
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+##
+trigger: none
+
+parameters:
+  - name: tag
+    displayName: 'Cargo Make Version Tag'
+    type: string
+  - name: python_version
+    displayName: 'Python Version'
+    type: string
+    default: '3.11'
```

Comment:
Not a big deal, but we're using 3.12 most other places so it might be nice to start there.

Reply:
switched to 3.12